### PR TITLE
Cloud neutral app

### PR DESCRIPTION
**Pull request by ChristophWurst**
This app works for ownCloud *and* Nextcloud, so let's remove the specific branding and make it a neutral app. This follows @oparoz'es PR on Gallery, see https://github.com/owncloud/gallery/pull/696.

@DeepDiver1975 @owncloud/mail objections?

### CODE REVIEW

**Comment by mar1u5** [comment]
:+1:

**Comment by mar1u5** [comment]
Shouldnt we also change the **README.md** to call it "Mail" instead of "ownCloud Mail" ❓

**Comment by ChristophWurst** [comment]
> Shouldnt we also change the README.md to call it "Mail" instead of "ownCloud Mail" :question:

Yes, of course. Pushed another commit

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mail",
   "version": "0.5.3",
-  "description": "ownCloud Mail App",
+  "description": "Mail App",
```
Comment (jancborchardt) [inline bower.json]: »Mail app«, app should be lowercase. ;)

File: appinfo/version [hunk 2]
```diff
@@ -1 +0,0 @@
-0.5.3
```
Comment (jancborchardt) [inline appinfo/version]: Why is this removed?

Comment (ChristophWurst) [inline appinfo/version]: AFAIK this file is not needed anymore

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mail",
   "version": "0.5.3",
-  "description": "Mail app for ownCloud",
+  "description": "Mail App",
```
Comment (jancborchardt) [inline package.json]: Also here, »app« lowercase please. ;)

File: package.json [hunk 4]
```diff
@@ -15,7 +15,6 @@
     "url": "https://github.com/owncloud/mail.git"
   },
   "keywords": [
-    "ownCloud",
     "mail"
   ],
   "author": "Thomas Müller and others",
```
Comment (jancborchardt) [inline package.json]: Author list could be updated too, to the one we did the other day.

File: appinfo/version (hunk 2 above)
Comment (ChristophWurst) [inline appinfo/version]: @MorrisJobke confirmed that it can be removed

File: tests/phpunit.xml [hunk 5]
```diff
@@ -5,7 +5,7 @@
 		 timeoutForMediumTests="900"
 		 timeoutForLargeTests="900"
 		>
-	<testsuite name='ownCloud - Mail App Tests'>
+	<testsuite name='Mail App Tests'>
```
Comment (jancborchardt) [inline tests/phpunit.xml]: »Mail app tests«. We only capitalize the first letters of the first word or of app names. ;)

**Comment by jancborchardt** [comment]
If you fix the lowercase stuff I commented you get a thumbs up ;)

**Comment by ChristophWurst** [comment]
comments fixed and conflicts resolved, please have another look @jancborchardt

**Comment by jancborchardt** [comment]
Thanks! :+1:
